### PR TITLE
TAUR-1238-Mobile-pipeline-saucelabs-issue

### PR DESCRIPTION
**Pull request by kanchan04katare**
In this PR, for taurus-android appium tests, I have added dependencies for executing methods. Using this we can reduce the time duration required for appium tests.

@jcasner @unixorn @lscheinkman @shantanoo ,  Please Review.

### CODE REVIEW

File: taurus-mobile/tests/behavioral/TaurusMobileApp/src/test/java/taurustest/TaurusMobileAppTest.java [hunk 1]
```diff
@@ -92,7 +93,7 @@ public void skipTutorial()
     }
 
 
-    @Test(priority = 1)
+    @Test(priority = 1,dependsOnMethods={"skipTutorial"})
```
Comment (shantanoo) [inline taurus-mobile/tests/behavioral/TaurusMobileApp/src/test/java/taurustest/TaurusMobileAppTest.java]: add space after comma.

File: taurus-mobile/tests/behavioral/TaurusMobileApp/src/test/java/taurustest/TaurusMobileAppTest.java [hunk 2]
```diff
@@ -111,7 +112,7 @@ public void addFavourites()
     }
 
 
-    @Test(priority = 2)
+    @Test(priority = 2 ,dependsOnMethods={"addFavourites"})
```
Comment (shantanoo) [inline taurus-mobile/tests/behavioral/TaurusMobileApp/src/test/java/taurustest/TaurusMobileAppTest.java]: s/ ,/, /
and
s/Methods=/Methods = /

File: taurus-mobile/tests/behavioral/TaurusMobileApp/src/test/java/taurustest/TaurusMobileAppTest.java [hunk 3]
```diff
@@ -133,7 +134,7 @@ public void removeFavourites()
         }
 
 
-    @Test(priority = 3)
+    @Test(priority = 3,dependsOnMethods={"removeFavourites"})
```
Comment (shantanoo) [inline taurus-mobile/tests/behavioral/TaurusMobileApp/src/test/java/taurustest/TaurusMobileAppTest.java]: s/3,d/3, d/
and
s/Methods=/Methods = /

File: taurus-mobile/tests/behavioral/TaurusMobileApp/src/test/java/taurustest/TaurusMobileAppTest.java [hunk 4]
```diff
@@ -148,7 +149,7 @@ public void clickOnCompanyName() throws InterruptedException {
     }
 
 
-    @Test(priority = 4)
+    @Test(priority = 4 , dependsOnMethods={"clickOnCompanyName"})
```
Comment (shantanoo) [inline taurus-mobile/tests/behavioral/TaurusMobileApp/src/test/java/taurustest/TaurusMobileAppTest.java]: s/4 ,/4,/
and
s/Methods=/Methods = /

**Comment by jcasner** [comment]
Still need to include maxDuration but otherwise good

**Pull request merged**